### PR TITLE
CLN: removed unused import

### PR DESCRIPTION
**Pull request by ShaharNaveh**
- [ ] closes #xxxx
- [ ] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

---

Not 100% sure about this, can close anytime if not a good PR

### CODE REVIEW

**Comment by MomIsBestFriend** [comment]
I also benched marked it:

master:

```
In [1]: import pandas as pd                                                                                   

In [2]: %timeit pd.Period("2000-01-01", freq="D")                                                             
165 µs ± 777 ns per loop (mean ± std. dev. of 7 runs, 10000 loops each)
```

---

PR:

```
In [1]: import pandas as pd                                                                                   

In [2]: %timeit pd.Period("2000-01-01", freq="D")                                                             
166 µs ± 449 ns per loop (mean ± std. dev. of 7 runs, 10000 loops each)
```

File: pandas/_libs/tslibs/np_datetime.pyx [hunk 1]
```diff
@@ -1,12 +1,15 @@
 from cpython.object cimport Py_EQ, Py_NE, Py_GE, Py_GT, Py_LT, Py_LE
 
-from cpython.datetime cimport (datetime, date,
-                               PyDateTime_IMPORT,
-                               PyDateTime_GET_YEAR, PyDateTime_GET_MONTH,
-                               PyDateTime_GET_DAY, PyDateTime_DATE_GET_HOUR,
```
Comment (jbrockmendel) [inline pandas/_libs/tslibs/np_datetime.pyx]: i find the year-month-day-hour-minute-second ordering clearer than the alphabetical ordering.  its not worth bikeshedding, which is why id discourage these for the most part

also makes it harder to see what the actual removed imports are (datetime and date, right?).  That part im on board with.

Comment (MomIsBestFriend) [reply pandas/_libs/tslibs/np_datetime.pyx]: > i find the year-month-day-hour-minute-second ordering clearer than the alphabetical ordering. its not worth bikeshedding, which is why id discourage these for the most part

Agreed, I have no idea how to enforce it with ```isort```

> also makes it harder to see what the actual removed imports are (datetime and date, right?). That part im on board with.

Correct, that's why I did it in two commits and not in one, as you an see after the removal of the unused imports what is left looks pretty ugly, so I figured I run ```isort``` on that specific import block.

**Review by WillAyd — APPROVED** [review]
Lgtm

**Comment by datapythonista** [comment]
Why is isort not complaining with the original code? Is it validating that file?

**Comment by MomIsBestFriend** [comment]
> Why is isort not complaining with the original code? Is it validating that file?

@datapythonista ```isort``` is not checking that file ATM, in the release of ```isort``` 5.0.0 it will begin to support ```.pyx``` files.

I think you ment ```flake8```, and I don't know why it's not catching it (or if it's even checking```.pyx``` files)

**Comment by WillAyd** [comment]
@datapythonista @jbrockmendel any outstanding objections?

**Review by datapythonista — APPROVED** [review]
lgtm

**Comment by jbrockmendel** [comment]
lgtm

**Pull request merged**